doc/modules/preprocessing.rst

```diff
@@ -413,28 +391,19 @@ This assumption is the base of the `Vector Space Model
 <https://en.wikipedia.org/wiki/Vector_Space_Model>`_ often used in text
 classification and clustering contexts.
 
-The function :func:`normalize` provides a quick and easy way to perform this
-operation on a single array-like dataset, either using the ``l1`` or ``l2``
-norms::
-
-  >>> X = [[ 1., -1.,  2.],
-  ...      [ 2.,  0.,  0.],
-  ...      [ 0.,  1., -1.]]
-  >>> X_normalized = preprocessing.normalize(X, norm='l2')
-
-  >>> X_normalized
-  array([[ 0.40..., -0.40...,  0.81...],
-         [ 1.  ...,  0.  ...,  0.  ...],
-         [ 0.  ...,  0.70..., -0.70...]])
-
-The ``preprocessing`` module further provides a utility class
-:class:`Normalizer` that implements the same operation using the
-``Transformer`` API (even though the ``fit`` method is useless in this case:
-the class is stateless as this operation treats samples independently).
+The :mod:`~sklearn.preprocessing` module provides a utility class
```

Comment:
We did not discuss that we could keep the `normalize` function since they are per sample and not per feature statistic and it does not really matter if one use the transformer or the function?

Reply:
You are absolutely right @glemaitre: https://github.com/scikit-learn/scikit-learn/issues/17402#issuecomment-640095189

Reply:
indeed let's not touch the Normalizer, section, neither the `Binarizer` section.

Though we'll need to update some parts like this one:

> As for the :class:`StandardScaler` and :class:`Normalizer` classes, the
preprocessing module provides a companion function :func:`binarize`

As @OlehKSS properly did already :+1:

Reply:
ok